RocketApp/Infrastructure/Sources/UIToolkit/LocalizedStringKey+keys.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+public extension LocalizedStringKey {
+  static let rockets: Self = "rockets".localizedStringKey
```

Comment:
Ty pouzivas tu `String` extension jenom tady? Tak to mi teda moc nedava smysl proc existuje... Navic manualne udrzovat lokalizacni klice neni uplne super. Ale pro ted asi "good enough". Ale zamysli se, jestli fakt davaji smysl ty `String+` funkce.

Reply:
Mame na to skript ktery to taha z google sheetu, ale kvuli tem par stringum to asi nebudu pouzivat. Funkce pouzivam jenom tady no, myslel jsem ze je to tak spravne uklizet je do extensions k prislusnym typum...